### PR TITLE
Update Batch preview module to support data plane API version 2019-08-01.10.0 and mgmt API version 2019-08-01

### PR DESCRIPTION
**Pull request by matthchr**
## Description

Update Batch preview module to support data plane API version 2019-08-01.10.0 and mgmt API version 2019-08-01

## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by matthchr** [comment]
This _does_ include breaking changes (on purpose).

The cmdlet review was done here: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/371

**Comment by markcowl** [comment]
@matthchr Looks like you need to update the online help version header in your new markdown files

**Comment by matthchr** [comment]
Thanks @markcowl - I've fixed that. I believe there may also be some breaking change detections I need to fix up, I will look

File: src/Batch/Batch/BatchAccounts/GetBatchAccountSupportedImagesCommand.cs [hunk 1]
```diff
@@ -19,8 +19,8 @@
 
 namespace Microsoft.Azure.Commands.Batch
 {
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzurePrefix + "BatchNodeAgentSku"),OutputType(typeof(PSNodeAgentSku))]
-    public class GetBatchAccountNodeAgentSkuCommand : BatchObjectModelCmdletBase
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzurePrefix + "BatchSupportedImages"),OutputType(typeof(PSImageInformation))]
```
[the remark this answers is not in the document]
Comment (markcowl) [reply src/Batch/Batch/BatchAccounts/GetBatchAccountSupportedImagesCommand.cs]: No, there is not. This PR will not be accepted without this change.

File: tools/StaticAnalysis/Exceptions/Az.Batch/SignatureIssues.csv [hunk 2]
```diff
@@ -111,4 +111,5 @@
...
-"Microsoft.Azure.PowerShell.Cmdlets.Batch.dll","Microsoft.Azure.Commands.Batch.NewBatchResourceFileCommand","New-AzBatchResourceFile","1","8100","New-AzBatchResourceFile Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
\ No newline at end of file
+"Microsoft.Azure.PowerShell.Cmdlets.Batch.dll","Microsoft.Azure.Commands.Batch.NewBatchResourceFileCommand","New-AzBatchResourceFile","1","8100","New-AzBatchResourceFile Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"D:\a\1\s\artifacts\Debug\Az.Batch\Microsoft.Azure.PowerShell.Cmdlets.Batch.dll","Microsoft.Azure.Commands.Batch.GetBatchAccountSupportedImagesCommand","Get-AzBatchSupportedImages","1","8400","Get-AzBatchSupportedImages uses the noun 'AzBatchSupportedImages', which does not follow the enforced naming convention of using a singular noun for a cmdlet name.","Consider using a singular noun for the cmdlet name."
```
Comment (markcowl) [inline tools/StaticAnalysis/Exceptions/Az.Batch/SignatureIssues.csv]: This is not allowed.

File: src/Batch/Batch/BatchAccounts/GetBatchAccountSupportedImagesCommand.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthchr) [reply src/Batch/Batch/BatchAccounts/GetBatchAccountSupportedImagesCommand.cs]: I believe I've made the required change

File: tools/StaticAnalysis/Exceptions/Az.Batch/SignatureIssues.csv (hunk 2 above)
Comment (matthchr) [reply tools/StaticAnalysis/Exceptions/Az.Batch/SignatureIssues.csv]: Removed

**Comment by matthchr** [comment]
It seems like the test-linux job is failing but it looks to be some internal build error to me...?

And the other 3 seem stuck?

**Comment by matthchr** [comment]
@markcowl - Hey Mark, I think this should be good to merge now based on your comments, but for some reason some of the PR gates seem stuck?

**Comment by matthchr** [comment]
Looks like the PR gates got sorted out

**Comment by matthchr** [comment]
@markcowl - Can we merge this now?

**Pull request merged**